item-descriptions.lua

```diff
@@ -463,124 +473,120 @@ descriptions = {
-                "trap or melted down to provide metal bars, redeeming them."},
+                "A halberd is an axe with a long haft and a sharp point that can be used",
+                "for stabbing attacks. It is 50% larger than the battle axe, making it",
+                "too large to be wielded by dwarves. As foreign weapons, they cannot be",
+                "forged by dwarves, either."},
     ITEM_WEAPON_HAMMER_WAR = {
                 "A war hammer is a blunt weapon that is essentially a hammer with a long",
-                "handle. War hammers use and train the hammerdwarf skill. Dwarves can",
+                "shaft. War hammers use and train the hammerdwarf skill. Dwarves can",
                 "forge war hammers out of any weapons-grade metal, though those with",
-                "higher densities tend to cause more damage."},
+                "higher densities tend to cause greater damage."},
     ITEM_WEAPON_MACE = {
                 "A mace is a blunt weapon that consists of a rounded or flanged weight",
-                "mounted on the end of a handle. Despite similarities to a morningstar",
-                "in appearance, a mace is 60% larger and has twice the contact area.",
-                "Maces use and train the macedwarf skill. Dwarves can forge maces out of",
-                "any weapons-grade metal, though those with higher densities",
-                "(like silver) cause more damage."},
+                "mounted on a shaft. Compared to the morningstar, a mace is 60% larger",
+                "and has twice the contact area. Maces use and train the macedwarf skill.",
+                "Dwarves can forge maces out of any weapons-grade metal, though those with",
+                "higher densities (like silver) have greater effectiveness."},
     ITEM_WEAPON_MAUL = {
                 "A maul is a blunt weapon that is essentially a very large war hammer,",
                 "similar to a sledgehammer. Mauls are more than three times larger than",
                 "standard war hammers, with a similar 'bash' attack. Mauls also have",
                 "ten times the contact area and greatly reduced penetration, which",
                 "reduces their effectiveness. Mauls use and train the hammerdwarf",
-                "skill. Being foreign weapons, dwarves cannot forge mauls."},
+                "skill. Being foreign weapons, mauls cannot be forged by dwarves."},
     ITEM_WEAPON_MORNINGSTAR = {
                 "A morningstar is an edged weapon that consists of a spiked ball mounted",
-                "on the end of a handle. Despite similarities to a mace in appearance,",
-                "a morningstar's size and contact area are closer to those of a war",
-                "hammer. Specifically, a morningstar is 25% larger than a war hammer",
+                "on the end of a shaft. Despite similarities to the mace in appearance,",
+                "a morningstar's size and contact area are closer to those of the war",
```

Comment:
"a" and "the" aren't used consistently

Reply:
Further cleaned up this and a lot of other things in many of the other weapon descriptions. I'm not sure what the grammatical term for this kind of article use is, but you can also see it in eg. [the Wikipedia article for the war hammer](https://en.wikipedia.org/wiki/War_hammer): "**A war hammer** is a late medieval weapon [...] whose design resembles **the hammer**."